code/lab1-tdd-arabic-roman-converter/src/main/java/ru/unn/agile/ArabicRomanConverter/ArabicToRomanConverter.java

```diff
@@ -0,0 +1,24 @@
+package ru.unn.agile.ArabicRomanConverter;
+
+public class ArabicToRomanConverter {
+
+    private static final int[] ARABIC_NUMERALS = {1000, 900, 500, 400,
+            100, 90, 50, 40, 10, 9, 5, 4, 1};
+    private static final String[] ROMAN_NUMERALS = {"M", "CM", "D", "CD",
+            "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"};
```

Comment:
Do we really need this? CM CD and others? In RomanToArabicConverter we have only 
private static final Character[] ROMAN_NUMERALS = {'I', 'V', 'X', 'L', 'C', 'D', 'M'};
and that's seems enough.

Reply:
In RomanToArabicConverter it's enough, but in this case it's easier to use such set . For example, when I convert 4 to "IV", I don't need to count "I" and replace it  by "IV". I can convert it immediately.